packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -214,23 +214,23 @@ export class EpochTracker {
                 // This will only throw if it is an epoch error.
                 this.checkForEpochErrorCore(epochFromResponse, error.errorMessage);
             } catch (epochError) {
-                const err = {
-                    ...epochError,
+                assert(epochError instanceof LoggingError);
+                epochError.addProperties({
                     fromCache,
                     clientEpoch: this.fluidEpoch,
                     fetchType,
-                };
-                this.logger.sendErrorEvent({ eventName: "EpochVersionMismatch" }, err);
+                });
+                this.logger.sendErrorEvent({ eventName: "EpochVersionMismatch" }, epochError);
                 assert(!!this.fileEntry, "File Entry should be set to clear the cached entries!!");
                 // If the epoch mismatches, then clear all entries for such file entry from cache.
                 await this.persistedCache.removeEntries(this.fileEntry);
                 throw epochError;
             }
-            // If it was categorised as epoch error but the epoch returned in response matches with the client epoch
+            // If it was categorized as epoch error but the epoch returned in response matches with the client epoch
             // then it was coherency 409, so rethrow it as throttling error so that it can retried. Default throttling
             // time is 1s.
             this.logger.sendErrorEvent({ eventName: "Coherency409" }, error);
```

Comment:
Is this a helpful log, or noisy/duplicate of the below thrown error which will presumably be logged later as well?

Reply:
So far it as been helpful, though in future we might decide to drop it. Ideally it should not happen that often (due to server having its own retry logic) and presence of it in logs immediately tells us something is off